docker-ui.Dockerfile

```diff
@@ -0,0 +1,51 @@
+# TODO: This comment might need to change, depending on how compose will work with turborepo
```

Comment:
Move to root. I couldn't get compose to work as expected with Dockerfiles in their respective project folders. This worked nicely.